tests/docs-screenshots.spec.ts

```diff
@@ -91,20 +111,617 @@ test("instances", async ({ page }) => {
+  await page.getByRole("link", { name: "Instances", exact: true }).click();
+  await page.getByText("Create instance").click();
+  await page.getByPlaceholder("Enter name").fill("Ubuntu-vm");
+  await page.getByRole("button", { name: "Browse images" }).click();
+  await page
+    .locator("tr")
+    .filter({ hasText: "Ubuntu24.04 LTSnoblealldefaultUbuntuSelect" })
+    .getByRole("button")
+    .click();
+
+  await page.getByLabel("Instance type").click();
+  await page.getByLabel("Instance type").selectOption("VM");
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/create_vm.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByLabel("Instance type").click();
+  await page.getByLabel("Instance type").selectOption("Container");
+  await page.getByPlaceholder("Enter name").fill(instance);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/create_instance.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByText("Resource limits").click();
+  await page.getByText("Resource limits").click();
+  await page
+    .getByRole("row", { name: "Exposed CPU limit" })
+    .getByRole("button")
+    .click();
+  await page.getByPlaceholder("Number of exposed cores").fill("1");
+  await page
+    .getByRole("row", { name: "Memory limit Usage limit" })
+    .getByRole("button")
+    .click();
+  await page.getByPlaceholder("Enter value").fill("4");
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/resource_limits.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByRole("button", { name: "Create and start" }).click();
+  await page.getByTestId("notification-close-button").click();
+
+  await page
+    .getByRole("row", {
+      name: `Select ${instance} default Name Type Description IPv4 Status Actions`,
+    })
+    .getByLabel("Name")
+    .click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/instance_summary.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await visitInstance(page, instance);
+  await page.getByTestId("tab-link-Configuration").click();
+  await page.getByText("Disk").click();
+  await page.getByRole("button", { name: "Attach disk device" }).click();
+  await page.getByRole("button", { name: "Mount host path" }).click();
+  await page
+    .getByPlaceholder("Enter full path (e.g. /home)")
+    .fill("default/tutorial_volume");
+  await page.getByPlaceholder("Enter full path (e.g. /data)").fill("/data");
+  await page.getByRole("button", { name: "Attach", exact: true }).click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/add_disk_device.png",
+    clip: getClipPosition(0, 0, 1440, 760),
   });
+
+  await page.locator("#form-footer span").first().click();
+  await page.waitForTimeout(1000);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/yaml_configuration.png",
```

Comment:
Need to wait a little longer, so the syntax highlighting kicks in

<img width="1920" height="972" alt="image" src="https://github.com/user-attachments/assets/d41dec6f-957a-4040-8f64-c4f03610dc62" />